Add deprecation note about CBC-based ciphers (#1962)
neo4j/docs-operations: modules/ROOT/pages/security/ssl-framework.adoc

Valid values depend on the current JRE and SSL provider.
For Oracle JRE here is the list of supported ones - https://docs.oracle.com/en/java/javase/17/docs/specs/security/standard-names.html#jsse-cipher-suite-names.

> [IMPORTANT]

Reviewer: ```suggestion
[IMPORTANT,role=label--deprecated-5.26]
```

Reviewer: or 
```
[role=label--deprecated-5.26]
[IMPORTANT]
```

====

> label:deprecated[Deprecated in 5.26]

Reviewer: ```suggestion
```

CBC (cipher block chaining) ciphers, as used in TLS v1.2 network encryption, have several security vulnerabilities that make them less secure than alternative methods.

> The Internet Engineering Task Force (IETF) doesn't recommend using CBC-based ciphers (RFC 8447), and these ciphers were removed from the TLS standard with the development of TLS v1.3.

Reviewer: ```suggestion
The Internet Engineering Task Force (IETF) does not recommend using CBC-based ciphers (RFC 8447), and these ciphers were removed from the TLS standard with the development of TLS v1.3.
```

> Note that the use of the following CBC-based ciphers has been deprecated in Neo4j 5.26:

Reviewer: ```suggestion
Note that the use of the following CBC-based ciphers are deprecated from Neo4j 5.26:
```
